pulpcore/cli/ansible/content.py

```diff
@@ -28,4 +50,96 @@ def content(ctx: click.Context, pulp_ctx: PulpContext, content_type: str) -> Non
         raise NotImplementedError()
 
 
-content.add_command(list_command())
+list_options = [
```

Comment:
These are really filter options, right? We could have gems like `--name-contains`...

Reply:
There is only exact filters on these fields.